pages/_app.js

```diff
@@ -1,6 +1,7 @@
 import { UserProvider } from "@auth0/nextjs-auth0";
 import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
 import appTheme from '../styles/theme';
+import '../styles/globals.css'
```

Comment:
Is this line needed? I don't see any changes in globals.css in this PR 🤔

Reply:
We were picking up 8px of margin from somewhere, I think it was the browser default which made me think that these globals weren't being respected. Importing them here means they are actually getting applied and we don't have that random margin creeping in

Reply:
Hmmm interesting, good stuff :)